store-implementation/proxy-store/src/test/java/uk/gov/gchq/gaffer/proxystore/operation/handler/GetProxyUrlHandlerTest.java

```diff
@@ -46,7 +46,7 @@ public void shouldGetURl() throws Exception {
         String expected = String.format("http://%s:%s/rest/v2", host, port, DEFAULT_GAFFER_CONTEXT_ROOT);
 
         //whenGetUrlOperationTest
```

Comment:
I presume this isn't changed as it will get fixed here: https://github.com/gchq/Gaffer/blob/87fcedc3b8a58fb5b2d439a039ceaa4312aec22d/store-implementation/proxy-store/src/test/java/uk/gov/gchq/gaffer/proxystore/operation/handler/GetUrlHandlerTest.java#L48

Reply:
Yeah if it gets done in one of them.